### PR TITLE
[MPR] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to PR https://github.com/reactos/reactos/pull/5908

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/win32/mpr/lang/mpr_Ro.rc [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
- *               2011 Fulea Ștefan
- *
+ * PROJECT:     ReactOS MPR
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2004 Juan Lang
+ *              Copyright 2008 Michael Stefaniuc
+ *              Copyright 2011 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
+
```
Comment (binarymaster) [inline dll/win32/mpr/lang/mpr_Ro.rc:9]: This is Wine file, do not modify the header: https://github.com/reactos/reactos/blob/bf95b7e8e5e7544a5048ed700fced44ba238d553/media/doc/WINESYNC.txt#L104

Comment (AndreiMiloiu) [reply dll/win32/mpr/lang/mpr_Ro.rc]: I understand. Is it good now?

**Review by JoachimHenze — APPROVED** [review]
Approved. No accelerator collisions.

File: dll/win32/mpr/lang/mpr_Ro.rc (hunk 1 above)
Comment (JoachimHenze) [reply dll/win32/mpr/lang/mpr_Ro.rc]: I would recommend to simply not add a ros-header-section here at all to elliminate the problem entirely.

Comment (AndreiMiloiu) [reply dll/win32/mpr/lang/mpr_Ro.rc]: I understand, but shouldn't I write my name somewhere?

File: dll/win32/mpr/lang/mpr_Ro.rc [hunk 2]
```diff
@@ -18,6 +18,16 @@
...
+/*
+ * PROJECT:     ReactOS MPR
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2004 Juan Lang
+ *              Copyright 2008 Michael Stefaniuc
+ *              Copyright 2011 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
+*/
+
```
Comment (binarymaster) [inline dll/win32/mpr/lang/mpr_Ro.rc:30]:
```suggestion
```
(this is Wine file, modify existing header without reformatting it)

File: dll/win32/mpr/lang/mpr_Ro.rc (hunk 1 above)
Comment (binarymaster) [reply dll/win32/mpr/lang/mpr_Ro.rc]: image

**Comment by AndreiMiloiu** [comment]
Thank you for help @binarymaster!

**Pull request merged**